src/index.html

```diff
@@ -1,13 +1,21 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Antisnake</title>
-  <link rel="stylesheet" href="./styles/main.scss">
-</head>
-<body>
-  <h1>Antisnake</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Antisnake</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+  </head>
+  <body>
+    <h1 hidden>Antisnake</h1>
+    <div class="blocks">
+      <div class="blocks__block blocks__block--1">1</div>
+      <div class="blocks__block blocks__block--2">2</div>
+      <div class="blocks__block blocks__block--3">3</div>
+      <div class="blocks__block blocks__block--4">4</div>
+      <div class="blocks__block blocks__block--5">5</div>
+      <div class="blocks__block blocks__block--6">6</div>
```

Comment:
```suggestion
      <div class="blocks__block">1</div>
      <div class="blocks__block">2</div>
      <div class="blocks__block">3</div>
      <div class="blocks__block">4</div>
      <div class="blocks__block">5</div>
      <div class="blocks__block">6</div>
```
Используй nth-child вместо дополнительных классов